grails-app/controllers/org/pih/warehouse/product/ProductTypeController.groovy

```diff
@@ -94,16 +94,10 @@ class ProductTypeController {
                     return
                 }
             }
-            if (params.supportedActivities) {
-                params.supportedActivities = params.list("supportedActivities") as ProductActivityCode[]
-            }
-            if (params.requiredFields) {
-                params.requiredFields = params.list("requiredFields") as ProductField[]
-            }
-            if (params.displayedFields) {
-                params.displayedFields = params.list("displayedFields") as ProductField[]
-            }
-            productTypeInstance.properties = params
+            params.supportedActivities = params.list("supportedActivities")
+            params.displayedFields = params.list("displayedFields")
+            bindData(productTypeInstance, params)
```

Comment:
what was causing the issue?

Reply:
The `<g:select />` tag did not send any data when no options were selected causing some issues when trying to clear all of the options